### DISCUSSION TITLE
Timezone configuration please.

### DISCUSSION THREAD

**Discussion by supr3m** [mockoon/mockoon#863] [Help] [answered]
It would be nice if the docker image can support timezone config with a TZ env.

In your image can use the `tzdata` package: https://stackoverflow.com/questions/68996420/how-to-set-timezone-inside-alpine-base-docker-image

docker/kubernetes would can:
```bash
containers:
   - name: mockoon
     image: mockoon/cli:latest
     args:
       - "--data"
       - "/path/to/mockoon-environment.json"
     env:
       - name: TZ
         # https://en.wikipedia.org/wiki/List_of_tz_database_time_zones#List
         value: "Foo"
```
¡Thanks!

**Comment by 255kb** [comment, marked as answer]
So, it's a matter of adding tzdata installation in the dockerfile? Then you would pass a TZ env var at runtime?

**Reply by supr3m** [reply]
For both questions, ¡yeap, that's it!

In fact, you can switch to another timezone in any runtime and not needed restart the container/pod.

**Reply by 255kb** [reply]
ok, 
Would you mind opening an issue so we can track this? 
Thank you!

**Reply by supr3m** [reply]
¡Done at https://github.com/mockoon/mockoon/issues/865!

**Reply by 255kb** [reply]
Thank you!
